Upgrade <textarea> to ember native component – like {{input}} already does

djsegal opened the pull request:
#### Commits in order:

----

1. 	Update textarea to use native ember helper

  + this literally uses the logic from https://github.com/piceaTech/ember-rapid-forms/commit/24cb1a7c46d21236d81bfe1ff9de6b3e48b1e5b6 to upgrade to ember native component


2.    Add `readonly` attribute to the two now that I know it is missing

3.    Add `autoresize` attribute to solve the original issue: #109 

---

GCorbel commented:
It seems to be nice. Very nice improvement. Can you just add attributes in the doc : http://piceatech.github.io/ember-rapid-forms/#/overview . Please, add other missing attributes.

djsegal commented:
@GCorbel: 
+ I just added stuff to the docs and rebased those changes into my commits
+ I don't think there are any more attributes worth adding at the moment.

Is there anything else you can think of? Thanks again!

GCorbel commented:
It looks good. Just wait the confirmation from @spruce

spruce commented:
👍
